test/format/config.h

```diff
@@ -54,7 +54,7 @@ typedef struct {
 	uint32_t	maxrand;	/* Maximum value randomly chosen */
 	uint32_t	maxset;		/* Maximum value explicitly set */
 	uint32_t	*v;		/* Value for this run */
-	char		**vstr;		/* Value for string options */
+	const char	**vstr;		/* Value for string options */
```

Comment:
Is there some benefit to this?  We allocate the memory, so we can write to it -- the only effect I see of this change is that we need to cast to `(void *)` in order to free...